app/webpacker/components/CompetitionsOverview/CompetitionsFilters.js

```diff
@@ -27,7 +27,8 @@ function CompetitionsFilters({
       <Form.Field>
         <EventSelector
           selectedEvents={filterState.selectedEvents}
-          dispatchFilter={dispatchFilter}
+          onEventSelection={dispatchFilter}
+          eventList={WCA_EVENT_IDS}
```

Comment:
Can you introduce this as a default value in `EventSelector`'s props or so?
The most common use-case is to be able to pick from ALL events (competition overview, personal records page, rankings pages, ...) so it feels redundant to always import a list of official events and pass it along.